lib/services/database_mutation_functions.dart

```diff
@@ -246,6 +254,30 @@ class DataBaseMutationFunctions {
     return null;
   }
 
+  Future<Beacon> changeBeaconDuration(String id, int newExpiresAt) async {
+    final QueryResult result = await clientAuth.mutate(
+      MutationOptions(
```

Comment:
Will this mutation update the expiry time for all joiners?

Reply:
Yes it changes the beacon duration, so when any joiner fetches beacon data it will have new duration.

Reply:
No other users will have to refresh to see the changes since backend does not provide with necessary subscriptions to do that atm. Will create a new issue for that.